doc/xplat_cli_migration.md

```diff
@@ -0,0 +1,223 @@
+# Migrating from Azure XPlat CLI to Azure CLI 2.0
+
+With the introduction of our Azure CLI 2.0 preview, customers now have a
+choice of Azure command-line tools.  While our Azure CLI 2.0 is built to be
+easier to use and automate, please consider the following before starting:
+
+* Both Azure CLI's can be installed and used side-by-side
+* The Azure CLI 2.0 will not support ASM/Classic mode services
+* Scripts are not compatible between both CLIs
+
+## Why consider trying the Azure CLI 2.0?
+Regardless of the tool or vendor, it is important to ensure you benefit when
+adding a new tool to your toolbox.  While the following covers only a small
+number of difference between these two products, we expect customers to benefit
+from the following:
+
+* Clean outputs for common workflows
+  * `--out table` for simplified human output
+  * `--out json` and `--out jsonc` for JSON outputs
+  * `--out tsv` for interacting with tools like grep, AWK, and jq
+* Improved and consistent in-tool documentation with `-h`
+  * Includes descriptions for groups, commands, and parameters
+  * Indicates where to look up required values
+  * Examples and links to web content
+* Improved command-line productivity
+  * Use `[tab][tab]` to lookup parameters, including resource groups and names (only supported in BASH and BASH on Windows)
+  * Work with either ID values (`--ids`) _or_ resource group and name (`-g -n`)
+  * Built in client-side query engine powered by JMESPath
+* (Coming) Support for more features and services
```

Comment:
>(Coming) Support for more features and services [](start = 2, length = 47)

this could just be something like "Service Support" since you have "coming soon" below